pennylane/ops/qutrit/__init__.py

```diff
@@ -0,0 +1,15 @@
+from .matrix_ops import *
+from ..identity import Identity
+
+ops = {
+    "Identity",
+    "QutritUnitary",
+}
+
+# TODO: Remove QutritUnitary from obs list
+obs = {
+    "QutritUnitary",        # Added here to prevent errors when using device
```

Comment:
It can't, as the list of observables must be limited to all observables defined inside the `/pennylane/ops/qutrit/` directory, and `Identity` is defined in `/pennylane/ops/`.